src/main/java/com/exactpro/epfast/template/implementations/AsciiStringFieldClass.java

```diff
@@ -0,0 +1,17 @@
+package com.exactpro.epfast.template.implementations;
```

Comment:
Rename package to `simple` and move to test sources. I don't see any use of that besides tests at the moment